src/Tribe/Rewrite.php

```diff
@@ -581,8 +588,11 @@ static function ( $rule_query_string ) use ( $pattern ) {
 		 * @param array $our_rules An array of rewrite rules handled by our code, in the shape
 		 *                         `[ <rewrite_rule_regex_pattern> => <query_string> ]`.
 		 *                         E.g. `[ '(?:events)/(?:list)/?$' => 'index.php?post_type=tribe_events&eventDisplay=list' ]`.
+		 * @param array<string,string> All the current rewrite rules, before any filtering is applied; these have the
+		 *                             same `<pattern => rewrite >` format as the previous argument, which is the
+		 *                             format used by WordPress rewrite rules.
 		 */
-		$our_rules = apply_filters( 'tribe_rewrite_handled_rewrite_rules', $our_rules );
+		$our_rules = apply_filters( 'tribe_rewrite_handled_rewrite_rules', $our_rules, $all_rules );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 100 characters

<sub>hash: acd05e4cf0929c8907aca2ca219a1cc5</sub>